Tests.Unit.Presentation.Web/Context/AuthenticationContextTest.cs

```diff
@@ -68,6 +70,41 @@ public void Invalid_userId_returns_zero()
             Assert.Equal(AuthenticationMethod.KitosToken, authContext.Method);
         }
 
+        [Theory]
+        [InlineData(1, true)]
+        [InlineData(2, false)]
+        public void Authenticated_User_Can_Have_Api_Access(int userId, bool apiAccess)
+        {
+            var authenticationContextFactory = new AuthenticationContextFactory(Substitute.For<ILogger>(), MakeMockContext(tokenAuth, "1", userId.ToString(), "true"), MakeMockUserRepository(apiAccess));
+
+            var authContext = authenticationContextFactory.Create();
+
+            Assert.Equal(userId, authContext.UserId);
+            Assert.Equal(apiAccess, authContext.HasApiAccess);
+        }
+
+        [Fact]
+        public void Unauthenticated_User_Can_Not_Have_Api_Access()
+        {
+            var authenticationContextFactory = new AuthenticationContextFactory(Substitute.For<ILogger>(), MakeMockContext(null, "invalid", "1", "false"), Substitute.For<IUserRepository>());
```

Comment:
Du har både gang i NSubstitute og Moq. Nej tak. Vælg den ene. I Moq kan du bruger Mock.Of<T> i stedet for Substitute.For...